pytripgui/canvas_vc/objects/ctx.py

```diff
@@ -22,6 +22,8 @@ def prepare_data_to_plot(self):
 
     def _set_aspect(self):
         if self.projection_selector.plane == "Transversal":
-            self.aspect = 1.0
-        else:
-            self.aspect = self.cube.slice_distance / self.cube.pixel_size
+            self.aspect = self.cube.dimx/self.cube.dimy
```

Comment:
The code looks the same in all classes, could it be moved to a mother class ?

Reply:
Superclass does not exist, but I can make one as a little refactor

Reply:
Done